modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -40,10 +40,17 @@ const get = (editor: Editor): Formats => {
         selector: 'img,audio,video',
         collapsed: false,
         styles: {
-          float: 'left'
+          float: 'left',
         },
         preview: 'font-family font-size'
       },
+      {
+        selector: '.mce-placeholder',
```

Comment:
Should these selectors be the custom element name then we don't need the extra class attribute on it? The class has the benefit of being generic so if we say have two types of placeholders they could share the same class and then get the same alignment behaviour. But that assumes that any new placeholder will also be a inline block type.

Reply:
I really wish we had some api to register these from the plugin side.

Reply:
I like the class/attribute approach because of that generic value. This way we don't have core aware of some plugin specific stuff. It is just a behavior that you can add in any plugin/placeholder element. 
If we add another not inline block placeholder we could have two different classes for alignment behavior like `.mce-placeholder-inline` and `.mce-placeholder-block`. Or even more generic ones for not just for placeholders